furnace/pid1.py

```diff
@@ -181,6 +184,14 @@ def run(self):
         logger.debug("Control pipe closed, stopping")
         return 0
 
+    # NOTE: use only before create_namespaces()
+    def _get_loop_devices(self):
```

Comment:
The rest of the code does not use protected methods, so I would have this one private too.

Reply:
And by that, I meant public. Sorry.